addons/account/models/partner.py

```diff
@@ -824,6 +824,7 @@ def _run_vat_test(self, vat_number, default_country, partner_is_company=True):
         :param vat_number: a string with the VAT number to check.
         :param default_country: a res.country object
         :param partner_is_company: True if the partner is a company, else False.
+
             .. deprecated:: 16.0
                 Will be removed in 16.2
```

Comment:
:eyes:

Reply:
It's not 16.2 yet, is it? :clown_face:

Reply:
It will be removed in https://github.com/odoo/odoo/pull/179078